channeldb/kvdb/etcd/bucket.go

```diff
@@ -11,9 +11,9 @@ const (
 )
 
 var (
-	bucketPrefix   = []byte("b")
-	valuePrefix    = []byte("v")
-	sequencePrefix = []byte("$")
+	valuePostfix   = []byte{0x00}
+	bucketPostfix  = []byte{0xFF}
```

Comment:
alternatively we could postfix the values to ensure that we can't have equivocating keys

Reply:
Just did a quick test and `bbolt` gives "incompatible value" error on clashes. Right now this wrapper allows keys and buckets with the same key (not checked/enforced). Got me thinking if this is wrong or not for *our* use cases. Prefixing/postfixing the value is an option too if we want to do this (as otherwise we need to fetch two keys to avoid the clash on every put and createbucket)

Reply:
After much consideration and offline discussion, made changes to make key collisions between buckets and values compatible with bbolt.